katcp/core.py

```diff
@@ -660,6 +661,9 @@ class ProtocolFlags(object):
     # New proposal flag to indicate that a device supports ?request-timeout-hint
     # See CB-2051
     REQUEST_TIMEOUT_HINTS = b'T'
+    # New proposal flag to indicate that a device supports configuring multiple
+    # sensors in a single ?sensor-sampling request
+    BULK_SET_SENSOR_SAMPLING = b'B'
```

Comment:
These comments should be removed - the flags are no longer "new proposals".  They have been accepted.